x/autopilot/keeper/airdrop_test.go

```diff
@@ -206,31 +248,72 @@ func (s *KeeperTestSuite) TestAirdropOnRecvPacket() {
 			airdropShouldUpdate:   false,
 		},
 		{
-			name:             "receiver too long",
-			forwardingActive: true,
+			name:                         "receiver too long",
+			autopilotClaimActive:         true,
+			autopilotClaimEnabledForHost: true,
+			destinationChannelID:         ibctesting.FirstChannelID,
+			destinationPortID:            transfertypes.PortID,
 			packetData: transfertypes.FungibleTokenPacketData{
 				Receiver: strings.Repeat("X", 300),
 				Memo:     "",
 			},
 			transferShouldSucceed: false,
 			airdropShouldUpdate:   false,
 		},
+		{
+			name:                         "not transfer channel",
+			autopilotClaimActive:         true,
+			autopilotClaimEnabledForHost: true,
+			destinationChannelID:         ibctesting.FirstChannelID,
+			destinationPortID:            "invalid_port",
+			packetData: transfertypes.FungibleTokenPacketData{
+				Receiver: getClaimPacketMetadata(strideAddress),
+				Memo:     "",
+			},
+			transferShouldSucceed: false,
+			airdropShouldUpdate:   false,
+		},
+		{
+			name:                         "invalid channel-id",
```

Comment:
can we add a case that uses an existing transfer channel that corresponse to a different host zone

Reply:
done